package.json

```diff
@@ -59,36 +59,36 @@
     "@types/async": "3.2.7",
     "@types/escape-html": "1.0.1",
     "@types/hosted-git-info": "3.0.2",
-    "@types/jest": "26.0.23",
-    "@types/lodash": "4.14.170",
+    "@types/jest": "26.0.24",
+    "@types/lodash": "4.14.171",
     "@types/ms": "0.7.31",
     "@types/numeral": "2.0.1",
     "@types/traverse": "0.6.32",
     "@types/truncate-utf8-bytes": "1.0.0",
-    "@typescript-eslint/eslint-plugin": "4.26.0",
-    "@typescript-eslint/parser": "4.26.0",
+    "@typescript-eslint/eslint-plugin": "4.28.4",
+    "@typescript-eslint/parser": "4.28.4",
     "doctoc": "2.0.1",
     "dotenv": "10.0.0",
-    "eslint": "7.30.0",
+    "eslint": "7.31.0",
     "eslint-config-algolia": "18.0.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-eslint-comments": "3.2.0",
     "eslint-plugin-import": "2.23.4",
     "eslint-plugin-jasmine": "4.1.2",
     "eslint-plugin-jest": "24.3.6",
-    "eslint-plugin-jsdoc": "35.4.1",
+    "eslint-plugin-jsdoc": "35.4.5",
     "eslint-plugin-prettier": "3.4.0",
     "eslint-plugin-react": "7.24.0",
-    "husky": "6.0.0",
+    "husky": "7.0.1",
```

Comment:
isn't this the one with the wild license?

Reply:
not sure what you are talking about, husky is MIT

Reply:
https://github.com/typicode/husky/issues/857 it was only v5